resources/assets/components/actions/SoftEdgeWidgetAction/SoftEdgeWidgetAction.js

```diff
@@ -25,10 +24,30 @@ class SoftEdgeWidgetAction extends React.Component {
    */
   loadSoftEdgeWidget = () => {
     const softEdgeId = this.props.softEdgeId;
+    let url = `//www.congressweb.com/dosomething/${softEdgeId}?acceptAuthor=true&memberId=${
+      this.props.userId
+    }`;
+
+    const prepopulatedFields = {
+      firstName: this.props.user.firstName,
+      lastName: this.props.user.lastName,
+      email: this.props.user.email,
+      mobile: this.props.user.mobile,
+      address: this.props.user.addrStreet1,
+      city: this.props.user.addrCity,
+      state: this.props.user.addrState,
+      zip: this.props.user.addrZip,
+    };
+
+    Object.keys(prepopulatedFields).forEach(key => {
+      if (prepopulatedFields[key] != null) {
+        url += `&${key}=${prepopulatedFields[key]}`;
+      }
+    });
 
     window.$cweb(() => {
       $cweb(`#congressweb-action-${softEdgeId}`).congressweb({
-        url: `//www.congressweb.com/dosomething/${softEdgeId}`,
+        url: `${url}`,
```

Comment:
Can this just be `url,` since it's already defined as a variable above?
```suggestion
        url,
```

Reply:
hmm @mendelB when I had it like that, it threw this error:
![Screen Shot 2019-05-06 at 4 57 15 PM](https://user-images.githubusercontent.com/9019452/57254760-10861d80-7020-11e9-889f-9abae8909805.png)

Reply:
Hmm did you add it as just `url` or `url: url`? (I believe the latter would trigger that error since the former is the convenient shorthand for such a scenario).

Reply:
a-ha! I did the `url: url` but just tried with `url` and it worked - thank you! Didn't know that was shorthand for that and good to know for the future!